matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/heartbeat/HeartbeatUnsubscribeRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.matchbook.sdk.core.clients.rest.dtos.heartbeat;
+
+import java.util.Collections;
+import java.util.Map;
+
+import com.matchbook.sdk.core.clients.rest.dtos.RestRequest;
+
+public class HeartbeatUnsubscribeRequest implements RestRequest {
+
+    @Override
+    public String resourcePath() {
+        return "v1/heartbeat";
+    }
+
+    @Override
+    public Map<String, String> parameters() {
+        return Collections.EMPTY_MAP;
```

Comment:
I believe it would be better to return Collections.emptyMap()

Reply:
Good point. Changed EMPTY_MAP to emptyMap()